lms/templates/learner_dashboard/program_details_sidebar.underscore

```diff
@@ -21,6 +21,7 @@
         </div>
     <% } %>
 </aside>
+<% if (!programDiscussionEnabled) { %>
```

Comment:
do you still need this check?

Reply:
is this template being shared between the new view and old view?

Reply:
@AhtishamShahid ^

Reply:
yes, it is shared between both views.